admin/adminActionsRestricted.php

```diff
@@ -1152,6 +1158,42 @@ public function showApiKeys($params)
 		";
 	}
 
+	public function calculateGR($params) {

```

Comment:
next line for { please

Reply:
Almost made it through a whole PR without an indentation/style issue. One of these times I will get it.

Reply:
Just put in a commit which addresses this.